app/livechat/server/api/v1/message.js

```diff
@@ -227,7 +228,9 @@ API.v1.addRoute('livechat/messages.history/:rid', {
 				limit = parseInt(this.queryParams.limit);
 			}
 
-			const messages = loadMessageHistory({ userId: guest._id, rid, end, limit, ls });
+			const messages = loadMessageHistory({ userId: guest._id, rid, end, limit, ls })
```

Comment:
The new implementation is returning an array of messages, but the original implementation needs to return an object containing a property called by `messages`, which will contain the array of messages.